solidity/KrakenPriceTicker.sol

```diff
@@ -4,36 +4,36 @@
    This contract keeps in storage an updated ETH/XBT price,
```

Comment:
Any changes to the `DieselPrice.sol` that also apply to other contracts (github url, changing `reverts` in `__callback`s to requires etc., etc.) here are hereby assumed - saving me having to rewrite them all again!